src/modules/trade/pure/UnlockWidgetScreen/index.tsx

```diff
@@ -0,0 +1,64 @@
+import SVG from 'react-inlinesvg'
+
+import iconCompleted from 'legacy/assets/cow-swap/check.svg'
+import { ButtonPrimary } from 'legacy/components/Button'
+import { ExternalLink } from 'legacy/theme'
+
+import * as styledEl from './styled'
+
+export type BulletListItem = {
+  id: number
+  content: string | React.ReactNode
+  isNew?: boolean
+}
+
+type UnlockWidgetProps = {
+  items: BulletListItem[]
+  handleUnlock: () => void
+  title: string
+  subtitle: string
+  buttonLink: string
+  orderType: string
+  buttonText: string
+}
+
+export function UnlockWidgetScreen({
+  handleUnlock,
+  buttonText,
+  orderType,
+  buttonLink,
+  items,
+  title,
+  subtitle,
+}: UnlockWidgetProps) {
+  return (
+    <styledEl.Container>
+      <styledEl.TitleSection>
+        <h3>{title}</h3>
+        <strong>{subtitle}</strong>
+      </styledEl.TitleSection>
+
+      {items && (
+        <styledEl.List>
+          {items.map(({ id, isNew, content }) => (
```

Comment:
I know this hasn't been introduced here, but anyway.

Do we really need the `id` property?
Can't we just get it from the list index?

Reply:
Updated to use index 👍